cou/utils/juju_utils.py

```diff
@@ -213,7 +220,9 @@ async def async_run_on_unit(unit_name, command, model_name=None, timeout=None):
     return _normalise_action_results(results)
 
 
-async def async_get_unit_from_name(unit_name, model=None, model_name=None):
+async def async_get_unit_from_name(
+    unit_name: str, model: Optional[Model] = None, model_name: Optional[str] = None
+) -> Unit:
     """Return the units that corresponds to the name in the given model.
 
     :param unit_name: Name of unit to match
```

Comment:
change `:type model: model.Model()` to `:type model: Optional[Model]`
change `:type model_name: string` to `:type model_name: Optional[str]`